src/benchq/vizualization_tools.py

```diff
@@ -0,0 +1,24 @@
+################################################################################
+# © Copyright 2022-2023 Zapata Computing Inc.
+################################################################################
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy as np
+
+
+def plot_graph_state_with_measurement_steps(
+    G, measurement_steps, cmap=plt.cm.rainbow, name="measurement_steps_plot"
```

Comment:
Please don't do one letter variables :)

Reply:
Sorry this is a standard name for a nx.Graph I can make it more specific.